### DISCUSSION TITLE
Issue creating Arrow Files w tutorial data

### DISCUSSION THREAD

**Discussion by libell1** [GreenleafLab/ArchR#1704] [Questions / Documentation]
Hello,

Trying to get familiar with ArchR by using the tutorial data. When I try to create the Arrow Files, only 2/3 are made (scATAC_BMMC_R1.arrow and scATAC_CD34_BMMC_R1.arrow)

I tried looking in the Questions/Discussion, but I do not think anyone else had a similar issue with the actual creation of the Arrow Files...

Log file: [ArchRLogs/ArchR-createArrows-a1164e4994b5-Date-2022-10-27_Time-11-58-20.log](url)

Thanks!

**Comment by rcorces** [comment]
Sorry, the link to your log file does not work for me. I would delete the folder where you downloaded the tutorial data and start from scratch. Perhaps there was a problem with your download.
